doc/projects.rst

```diff
@@ -2,29 +2,28 @@
 Projects
 ########
 
-Spyder allows users to associate a given directory with a **Project**, which offers several main advantages:
+Spyder allows you to associate a given directory with a **Project**, which automatically saves and restores your :doc:`editor` panes and open files to exactly how you left them in your last session.
```

Comment:
```suggestion
Spyder allows you to associate a given directory with a **Project**, which automatically saves and restores your :doc:`editor` open files to exactly how you left them in your last session.
```

I don't think it's necessary to mention the editor panes at this point.

Reply:
In fact, the wording is incorrect (should be panels, not panes); probably my original mistake. Revising the text some accordingly (still not sure I 100% like it, but eh):

```suggestion
Spyder allows you to associate a given directory with a **Project**, which automatically saves and restores the files you have open in the :doc:`editor` from the last time you opened that project.
```